game_catalog/tests/test_models.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+from game_catalog.models import Comment
+
+pytestmark = pytest.mark.django_db
+
+
+def test_genre_str(genre):
+    assert str(genre) == "Action"
+
+
+def test_studio_str(studio):
+    assert str(studio) == "Epic Games"
+
+
+def test_game_str(game):
+    assert str(game) == "Fortnite"
+
+
+def test_user_is_favorite(user, game):
+    user.favorite_games.add(game)
+    assert user.is_favorite(game) is True
```

Comment:
```suggestion
    assert user.favorited(game) is True
```
I guess would be sligthly more concise, but its up to you.
Enuse if you change name to make it acroos project.

Reply:
Done

c828f0ce0f04b621429947330e5da0ce086faf7a